dotcom-rendering/src/client/sentryLoader/loadSentry.ts

```diff
@@ -63,20 +66,22 @@ const loadSentryOnError = (): void => {
 			// send any queued errors
 			while (queue.length) {
 				const queuedError = queue.shift();
-				if (queuedError) reportError(queuedError);
+				if (queuedError) reportError(queuedError, feature);
 			}
 			log('dotcom', `Injected Sentry in ${endPerformanceMeasure()}ms`);
 		};
 
 		// This is how we lazy load Sentry. We setup custom functions and
 		// listeners to inject Sentry when an error happens
 		window.onerror = (message, url, line, column, error) =>
-			injectSentry(error);
+			injectSentry(error, 'unknown');
 		window.onunhandledrejection = (
 			event: undefined | { reason?: unknown },
-		) => event?.reason instanceof Error && injectSentry(event.reason);
-		window.guardian.modules.sentry.reportError = (error) => {
-			injectSentry(error).catch((e) =>
+		) =>
+			event?.reason instanceof Error &&
+			injectSentry(event.reason, 'unknown');
+		window.guardian.modules.sentry.reportError = (error, feature) => {
```

Comment:
@SiAdcock Sorry if I'm missing something but this is still missing the custom tags?